backend/bitboxbase/updater/updater.go

```diff
@@ -38,6 +38,10 @@ type Updater struct {
 	blockchainInfo string
 	log            *logrus.Entry
 	running        bool
+	ip             string
+	rootURL        url.URL
```

Comment:
the base of all the urls will be the same, and the endpoints are constants (the api is not dynamic), so it's enough to just keep the ip here and put the endpoints into constants.